test/libyul/objectCompiler/eof/auxdata_load_store.yul

```diff
@@ -0,0 +1,59 @@
+object "a" {
+    code {
+        {
+            auxdatastore(0, 0, 0x1122334455667788990011223344556677889900112233445566778899001122)
+            returncontract("b", 0, 32)
+            return(0, 32)
```

Comment:
```suggestion
            returncontract("b", 0, 32)
```

Reply:
BTW. This reminds me that `returncontract` cannot continue. Does yul verify that there are not unreachable instructions?

Reply:
Not sure actually. Probably not. We have a warning at Solidity level but Yul analysis is very simple so I wouldn't be surprised if we don't detect it. I did a quick check by compiling a contract and I did not get such a warning.

Unreachable Yul code is eventually removed by the optimizer anyway.